code/modules/spells/spell_types/knock.dm

```diff
@@ -30,3 +30,13 @@
 /obj/effect/proc_holder/spell/aoe_turf/knock/proc/open_closet(var/obj/structure/closet/C)
 	C.locked = FALSE
 	C.open()
+
+/obj/effect/proc_holder/spell/aoe_turf/knock/poorcast
+	name = "Unskilled Knock"
+
+/obj/effect/proc_holder/spell/aoe_turf/knock/poorcast/cast(list/targets,mob/user = usr)
+	var/mob/living/carbon/poor = user
+	if(prob(20))
+		poor.adjustBruteLoss(10)
```

Comment:
Have it force a scream as well, some form of player feedback that isn't purely in the chat